### ISSUE TITLE
Mapped Network Drives Still Accessible Inside Sandbox

### ISSUE THREAD

**Issue by vrxgaminginc** [sandboxie-plus/Sandboxie#2629]
### Describe what you noticed and did

1. Map Network Drive (Y:\) to a Remote System
1. Create a new Hardened Sandbox with Data Protection
2. Edit INI

`BlockNetworkFiles=y`
`AllowNetworkAccess=<BlockNetAccess>,n`

4. Launch Windows Explorer from Sandboxie

EXPECT: 
Y:\ Should not be accessible

ACTUAL:
Y:\ is accessible

### How often did you encounter it so far?

always

### Affected program

No relevant

### Download link

Not relevant

### Where is the program located?

Not relevant to my request.

### Expected behavior

Y:\ Should not be accessible

### What is your Windows edition and version?

Win 10 Pro

### In which Windows account you have this problem?

A local or Microsoft account without special changes.

### Please mention any installed security software

none

### What version of Sandboxie are you running?

1.6.6

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

In a hardened sandbox with data protection (red sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### Sandboxie.ini configuration

```shell
Enabled=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#0423ee,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=9
AutoRecover=y
UseSecurityMode=y
UsePrivacyMode=y
BlockNetworkFiles=y
AllowNetworkAccess=<BlockNetAccess>,n
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
```

**Comment by offhub** [comment]
I can confirm this issue. The issue occurs if the `RestrictDevices=y` (or `UseSecurityMode=y`) option is enabled.

https://user-images.githubusercontent.com/6871698/214565122-f5d6e89a-68eb-4120-a3d0-696efe0f2880.mp4

**Comment by DavidXanatos** [comment]
will be fixed in 1.7.1

**Comment by DavidXanatos** [comment]
the fix is already commited to github, but will only be active in release 1.7.1 as its in the driver which will need signing, the live preview builds 1.7.0a,b,c,... all use the 1.6.7 driver hence the fix will not be available in hose
